### PR TITLE
adding usage images

### PR DESCRIPTION
**Pull request by designertyler**
Added new images for notification usage and edited markdown.

### CODE REVIEW

File: src/content/components/notification/usage.md [hunk 1]
```diff
@@ -27,6 +27,15 @@ We recommend that toast notifications automatically disappear after five seconds
...
 
+<image-component cols="8">
+
+![The three icon states](images/notification-usage-1.png)
+
+</image-component>
+
+## Placement
+
+
```
Comment (marijohannessen) [inline src/content/components/notification/usage.md]: It looks like there are two H2's next to each other here, is that intentional?

**Comment by jeanservaas** [comment]
quick comment:

I know that one slider image is an animation and one is static, but just wondering why they're not the same height and also why one goes full screen and one is short -- I like the proportions and size of the shorter one better, wondering if they could just both look like that?

**Comment by designertyler** [comment]
@jeanservaas Fixed the animated one to be 8 columns like the static image.

File: src/content/components/notification/usage.md (hunk 1 above)
Comment (designertyler) [reply src/content/components/notification/usage.md]: There was a duplicate `## Placement` in the wrong spot. I deleted the extra.

**Pull request merged**